nahusenayElias/Team_4
# DRD-76: Configure trusted host settings

## Related ticket
[DRD-76](https://edu-team4-react24k.atlassian.net/jira/software/projects/DRD/boards/1?selectedIssue=DRD-76)

## In this PR: 
- Added configuration to web/sites/default/settings.php to add localhost as a trusted host to avoid errors/warnings on Drupal
- Updated core 

## Testing instructions: 
- checkout branch
- lando composer install
- Check Drupal > Reports > Status report that there are no errors 

[DRD-76]: https://edu-team4-react24k.atlassian.net/browse/DRD-76?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
